extensions/omni_httpd/tests/limited_max_workers.yml

```diff
@@ -0,0 +1,18 @@
+$schema: "https://raw.githubusercontent.com/omnigres/omnigres/master/pg_yregress/schema.json"
+instance:
+  config:
+    shared_preload_libraries: */env/OMNI_SO
+    max_worker_processes: 3
+  init:
+  - create extension omni_httpd cascade
+  - create extension omni_httpc cascade
+
+tests:
+- name: should respond successfully
+  query: |
+    select
+      status
+    from
+      omni_httpc.http_execute(omni_httpc.http_request('http://localhost:8080'));
+  results:
+    - status: 200
```

Comment:
This returns `null` randomly for me locally. Maybe I'm missing some type of `wait` signal.

Reply:
It is an annoying known issue.

Currently solved in other tests with something like:

```yaml
- call omni_httpd.wait_for_configuration_reloads(2)
```

in the `init` section.

Basically, it's a timing issue – we are trying to test things before omni_httpd is able to ingest the new configuration, which is currently done post-transaction using LISTEN/NOTIFY. The waiting hack is ensuring the configuration has been reloaded. (2) is because of, well, how many times this happens in our current implementation. 

I have not spent much time fixing this anticipating work on the updated workers architecture.

Reply:
Weirdly enough it was the `localhost`, with that I got `error: connection refused` randomly. 

That happens even with `omni_httpd.wait_for_configuration_reloads(2)`.

For now I've just changed `localhost` to `127.0.0.1`.

Reply:
I might be aware of this `localhost` issue, too! It has to do with h2o's resolver, which would randomly pick whatever host would resolve to. If your localhost also resolves to an IPv6 address, for example, and we're not listening there, it won't connect.